lua/ide/panels/panel.lua

```diff
@@ -12,6 +13,35 @@ Panel.PANEL_POSITIONS = {
     Panel.PANEL_POS_TOP, Panel.PANEL_POS_LEFT, Panel.PANEL_POS_RIGHT, Panel.PANEL_POS_BOTTOM
 }
 
+local bool_to_string = function(b)
+    return b and "true" or "false"
+end
+
+local debug_winsize = false
+local winsize_debug = function(log, msg)
```

Comment:
Now that you've discovered the issue, can you remove this logging? I don't think the flag `debug_winsize` should belong here, and we probably don't want to do all those `vim.api` calls when we open the panel, if the flag is removed.

Reply:
Done.